### DISCUSSION TITLE
유니티 에디터에서 플레이시 카메라 회전 제스쳐 미작동

### DISCUSSION THREAD

**Discussion by kyn320** [naverz/zepeto-studio-kor#821] [5. Q&A]
![GIF 2022-05-30 오후 6-39-53](https://user-images.githubusercontent.com/11557205/170964703-6b7539cf-a0e3-427c-8660-f72809e5ec09.gif)

에디터에서 제페토 월드 실행시, 타 제스처 (이동, 점프, 카메라 줌 인/아웃)은 모두 정상 작동되는데, 카메라 회전만 작동이 안되고 있습니다.
제가 확인해야 할 사항이 있을까요?

**Comment by Astoration** [comment]
기본 컨트롤러 프리팹에서 무브 패드가 지정된 영역을 확인해주세요 지금 패드가 카메라 컨트롤영역을 덮어서 패드가 우선되는거 같네요
